buildSrc/src/main/kotlin/com/datadog/gradle/plugin/verification/GenerateVerificationXmlTask.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.verification
+
+import com.datadog.gradle.config.AndroidConfig
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import org.redundent.kotlin.xml.PrintOptions
+import org.redundent.kotlin.xml.xml
+import java.io.File
+import java.security.MessageDigest
+
+open class GenerateVerificationXmlTask : DefaultTask() {
```

Comment:
should we have a test/validation that generated XML indeed matches the schema?

Reply:
That's a good point, I'll try and see if I can write a test for that 👍